app/controllers/titles_controller.rb

```diff
@@ -4,7 +4,12 @@ class TitlesController < ApplicationController
   before_action :set_title, only: %i[show customer_resources]
 
   def index
-    @titles = titles.all(q: params[:q], page: params[:page])
+    @titles = titles.all(
+      q: params[:q],
+      page: params[:page],
+      filter: params[:filter]
```

Comment:
i wonder if there's a neater way to do this, like with ActionController params or something.  just a thought, this is fine with me as-is.